examples/main.go

```diff
@@ -195,32 +193,32 @@ func (c *Counter) HandleEvent(ctx context.Context, e *live.Event) error {
 	return nil
 }
 
-func (c *Counter) Render(ctx context.Context, meta *live.Meta) *htmltmpl.Template {
+func (c *Counter) Render(ctx context.Context, meta *live.Meta) (*htmltmpl.Template, any) {
```

Comment:
I might look cleaner to do `any, *htmltmpl.Template`, as the template is more likely to be the larger number of lines